.github/workflows/esp32-mkimage.yaml

```diff
@@ -112,3 +109,26 @@ jobs:
         name: atomvm-${{ matrix.soc }}-image
         path: ./src/platforms/esp32/build/atomvm-${{ matrix.soc }}-*.img
         if-no-files-found: error
+
+  release-if-master:
+    if: github.ref_name == 'master'
+    needs: build-esp32-img
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Download artifacts"
+        uses: actions/download-artifact@v3
+
+      - name: Display structure of downloaded files
+        run: ls -R
+
+      - name: Upload latest release
+        uses: pyTooling/Actions/releaser@main
+        with:
+          tag: latest
```

Comment:
I was thinking about this again, shouldn't be better "snapshot" instead of "latest"?
latest usually means, latest released release.